frontend/src/metabase/query_builder/components/view/View/NotebookContainer/NotebookContainer.tsx

```diff
@@ -124,7 +124,7 @@ export const NotebookContainer = ({
           style={{ flex: 1, overflowY: "auto" }}
         >
           <Notebook
-            question={question}
+            question={question.setType("question")}
```

Comment:
shouldn't we check here, if the question is metric and only then set it's type? maybe I don't understand this change

Reply:
This component is rendered only for questions. Unfortunately, until the first query change the underlying type would be wrong. I'll try to address this separately.